RELEASE_PROCESS.rst

```diff
@@ -0,0 +1,121 @@
+Prepare for the release
+-----------------------
+
+Clone a fresh copy from the root repo. Do not attempt a release in your
+current working repository as the following commands expect a fresh clone.
+
+::
+
+     git clone git@github.com:juju/theblues.git
+     cd theblues
+
+
+Find the next release number. You can check the last release by running `git
+tag` and finding the largest version number. Note that the values are sorted
+as strings, so don't just look at the last value.
```

Comment:
So, it's worth noting that right now we don't have a tagged version--we'll want to find the 0.0.1 sha and tag that for this to be accurate.